.github/workflows/moodle-ci.yml

```diff
@@ -28,28 +28,35 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - php: 8.0
-            moodle-branch: MOODLE_311_STABLE
-            database: pgsql
-            suite: default
-            profile: chrome
-          - php: 7.3
-            moodle-branch: MOODLE_311_STABLE
+          - php: 8.1
+            moodle-branch: MOODLE_401_STABLE
             database: mariadb
             suite: classic
             profile: default
+          - php: 8.1
+            moodle-branch: MOODLE_401_STABLE
+            database: pgsql
+            suite: default
+            profile: chrome
     steps:
       - name: Check out repository code
         uses: actions/checkout@v3
         with:
           path: plugin
 
+      - name: Install node
```

Comment:
Why are you adding here this manual node installation? The "Initialise moodle-plugin-ci" step below does that automatically, installing the CORRECT node version (from .`nvmrc`file) that, for MOODLE 401_STABLE is `lts/gallium -> v16.20.2`

Reply:
Note that the node think is going to be fixed by #776, so not relevant to this PR any more.

Same applies to the classic failures, fixed by #777

Reply:
Thank you for reminding this!